src/app/views/map/category-dropdown.js

```diff
@@ -0,0 +1,38 @@
+define(['jquery', 'underscore', 'backbone', 'text!templates/map_controls/category-dropdown.html',], 
+  function($, _, Backbone, MapCategoryDropdownTemplate){
+    var CategoryDropdownView = Backbone.View.extend({
+      $container: $('#toolbar'),
+      className: 'toolbar-control-container',  
+
+      initialize: function(state) {
+        this.state = state;
+        this.render();
+      },
+
+      render: function() {
+        this.$el.appendTo(this.$container);
+        const template = _.template(MapCategoryDropdownTemplate);
+        this.$el.html(template({
+          content: "fas fa-bars"
```

Comment:
is it ever anything other than `fa-bars`? If not, no need for a template and we can avoid this interpolation... just inline the SVG into the template file.